catalog/datasets/CM2.6 2xCO2.yaml

```diff
@@ -0,0 +1,23 @@
+name: "CM2.6 2xCO2"
+description: "GFDL CM2.6 2xCO2 Ocean Simulation" 
+stores:
+  - name: "GFDL_CM2_6_CONTROL_5-DAY_FIELD2023-05-13 13:34:59.702232.zarr"
+    href: "gs://leap-persistent-ro/groundpepper/GFDL_cm2.6_2xCO2/GFDL_CM2_6_CONTROL_5-DAY_FIELD2023-05-13 13:34:59.702232.zarr"
+  - name: "GFDL_CM2_6_CONTROL_DAILY_100M2023-05-13 17:29:58.276652.zarr"
+    href: "gs://leap-persistent-ro/groundpepper/GFDL_cm2.6_2xCO2/GFDL_CM2_6_CONTROL_DAILY_100M2023-05-13 17:29:58.276652.zarr"
+  - name: "GFDL_CM2_6_CONTROL_DAILY_SURF2023-05-13 02:17:45.252382.zarr"
+    href: "gs://leap-persistent-ro/groundpepper/GFDL_cm2.6_2xCO2/GFDL_CM2_6_CONTROL_DAILY_SURF2023-05-13 02:17:45.252382.zarr"
+  - name: "GFDL_CM2_6_CONTROL_MONTHLY_ADVTERM2023-05-13 07:04:55.441805.zarr"
+    href: "leap-persistent-ro/groundpepper/GFDL_cm2.6_2xCO2/GFDL_CM2_6_CONTROL_MONTHLY_ADVTERM2023-05-13 07:04:55.441805.zarr"
+  - name: "GFDL_CM2_6_CONTROL_MONTHLY_UPPER100M2023-05-13 19:02:29.175889.zarr"
+    href: "gs://leap-persistent-ro/groundpepper/GFDL_cm2.6_2xCO2/GFDL_CM2_6_CONTROL_MONTHLY_UPPER100M2023-05-13 19:02:29.175889.zarr"
```

Comment:
The `'name'` entry is the 'display' name. Can you format this a bit nicer to read (e.g. spaces instead of underscores, maybe a slightly shorter and more descriptive title?